pkg/resource/events-logger-secret/grafana-agent-secret.go

```diff
@@ -22,26 +21,14 @@ type extraSecret struct {
 	Data map[string]string `yaml:"data" json:"data"`
 }
 
-// SecretMeta returns metadata for the grafana-agent-secret
-func SecretMeta(lc loggedcluster.Interface) metav1.ObjectMeta {
-	metadata := metav1.ObjectMeta{
-		Name:      getGrafanaAgentSecretName(lc),
-		Namespace: lc.GetAppsNamespace(),
-		Labels:    map[string]string{},
-	}
-
-	common.AddCommonLabels(metadata.Labels)
-	return metadata
-}
-
 // GenerateGrafanaAgentSecret returns a secret for
 // the Loki-multi-tenant-proxy config
-func GenerateGrafanaAgentSecret(lc loggedcluster.Interface, credentialsSecret *v1.Secret, lokiURL string) (v1.Secret, error) {
+func GenerateGrafanaAgentSecret(lc loggedcluster.Interface, credentialsSecret *v1.Secret, lokiURL string) (map[string][]byte, error) {
```

Comment:
Why that specific change?

Reply:
To stay compliant with the other functions (the one for alloy and the one for promtail) so that I can keep the same piece of code in the `events-logger-secret.go` file